spec/integration/transport/local_spec.rb

```diff
@@ -69,7 +69,8 @@
           expect(runner.upload(target, file.path, dest, run_as: user).message).to match(/Uploaded/)
           expect(runner.run_command(target, "cat #{dest}", run_as: user)['stdout']).to eq(contents)
           expect(runner.run_command(target, "stat -c %U #{dest}", run_as:  user)['stdout'].chomp).to eq(user)
-          expect(runner.run_command(target, "stat -c %G #{dest}", run_as:  user)['stdout'].chomp).to eq(user)
+          expect(runner.run_command(target, "stat -c %G #{dest}", run_as:  user)['stdout'].chomp)
+            .to eq(user) | eq('docker')
```

Comment:
What's the condition where this is docker?

Reply:
The runner's primary group is `docker` and it can't be changed.

Reply:
Will it ever be 'runner' then?

Reply:
No. But if these tests are run locally, then the group should match the user.

Reply:
I don't think these tests will get executed locally, since you'd need to set the 'BOLT_SUDO_USER' env var in order to run tests tagged 'sudo', which I don't think we document anywhere or need to do unless we're developing something specific locally. Even if you do set that to true locally, you'd need to have the 'runner' user be the one executing the rspec tests, and I don't think we set up that infra anywhere? Basically, I think we can just check for the 'docker' group.

Reply:
Ah, got it 👍